SCClassLibrary/Common/Control/Server.sc

```diff
@@ -786,18 +786,18 @@ Server {
 		condition.wait
 	}
 
-	waitForBoot { |onComplete, limit = 100, onFailure|
+	waitForBoot { |onComplete, limit = 100, onFailure, clock = \AppClock|
```

Comment:
remove `clock = \AppClock` for `clock`.

Reply:
Thank you, and sorry for forgetting. Done! https://github.com/supercollider/supercollider/pull/6198/commits/cb686efe3a28162bf8a55a4aef44c1126967243f